modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step10_rgmb.md

```diff
@@ -0,0 +1,143 @@
+  - Don't want explicitly defined pins in your assembly? See [step11_rgmb_homo.md] for how to define homogenized assemblies with [AssemblyMeshGenerator.md].
+
+!media tutorial04_meshing/rgmb_assemblymesh_hex.png
+       id=tutorial04-rgmb_assemblymesh_hex
+       caption=Hexagonal assembly colored by `subdomain_id` (left), `region_id` (middle), and `pin_id` (right). Subdomain ids categorize elements as belonging to different subdomains (blocks), region ids categorize elements belonging to different regions (material zones), and `pin_ids` categorize elements belonging to different pin cells.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_hexagonal.i
+         id=tutorial04-rgmb_rgmb_core_hexagonal
+         caption=Example RGMB hexagonal assembly.
+         block=Mesh/assembly1
+
+## CoreMeshGenerator
+
+[CoreMeshGenerator.md] calls [PatternedHexMeshGenerator.md] or [PatternedMeshGenerator.md] to generate a Cartesian or hexagonal lattice of assembly-like structures.
+
+Defining `dummy_assembly_name` and using it in inputs defines a dummy assembly in the core
+
+### Cartesian Core Example
+
+!media tutorial04_meshing/rgmb_coremesh_cart.png
+       id=tutorial04-rgmb_coremesh_cart
+       caption=Cartesian core colored by `subdomain_id` (left), `region_id` (middle), and `assembly_id` (right).
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_cartesian.i
+         id=tutorial04-rgmb_core_cart-core
+         caption=Example RGMB Cartesian core.
+         block=Mesh/rgmb_core
+
+### Hexagonal Core Example
+
+!media tutorial04_meshing/rgmb_coremesh_hex.png
+       id=tutorial04-rgmb_coremesh_hex
+       caption=Hexagonal core colored by `subdomain_id` (left), `region_id` (middle), and `assembly_id` (right).
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_hexagonal.i
+         id=tutorial04-rgmb_core_hex-core
+         caption=Example RGMB Hexagonal core.
+         block=Mesh/rgmb_core
+
+## CoreMeshGenerator with Peripheral Ring
+
+A core periphery region can be added utilizing either the [PeripheralRingMeshGenerator.md] or the [PeripheralTriangleMeshGenerator.md].
+
+!media tutorial04_meshing/rgmb_periphery.png
+       id=tutorial04-rgmb_periphery
+       caption=Hexagonal core with core periphery meshed.
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_hexagonal_periphery.i
+         id=tutorial04-rgmb_core_hex-periphery
+         caption=Example RGMB Hexagonal core with periphery.
+         block=Mesh/rgmb_core
+
+!content pagination previous=tutorial04_meshing/step09_hpmr.md
+                    next=tutorial04_meshing/step11_rgmb_homo.md
```

Comment:
```suggestion
                    next=tutorial04_meshing/step11_rgmb_homo.md
                    
                    
## Default Block and External Boundary Names
An RGMB case can terminate at the pin, assembly, or core "level". Pins and assemblies always have a "type" assigned (a numeric integer), whereas cores do not. The "level" and "type" are used in the naming schemes for the blocks and outer boundary as follows.

The default block names in the final mesh are RGMB_(level)(type) and RGMB_(level)(type)_TRI (if triangular elements are present). For example, RGMB_PIN1 and RMGB_PIN1_TRI, or RGMB_ASSEMBLY1 and RGMB_ASSEMBLY1_TRI, or RGMB_CORE, for workflows ending at the pin, assembly or core level, respectively. 

The default outer boundary names in the radial dimension are "outer_(level)_(type)". For example, for workflows ending at the pin, assembly or core level respectively, the outer boundary names are "outer_pin_1", "outer_assembly_1" and "outer_core". If the problem is extruded in the axial dimension, the  "top" and "bottom" boundaries also exist.
```


@shikhar413  - please review/approve

Reply:
@eshemon the term `type` might be confusing to users. I think we should indiciate which parameter controls the type parameter (i.e. assembly_type for assemblies and pin_type for pins). Also RGMB_CORE_TRI can be a valid block as well

Suggested update:

## Default Block and External Boundary Names

An RGMB case can terminate at the pin, assembly, or core "level". Pins and assemblies always have a "type" assigned (a numeric integer specified as [!param](/Mesh/PinMeshGenerator/pin_type) for pins and [!param](/Mesh/AssemblyMeshGenerator/assembly_type) for assemblies), whereas cores do not. The "level" and "type" are used in the naming schemes for the blocks and outer boundary as follows.
The default block names in the final mesh are `RGMB_(level)(type)` and `RGMB_(level)(type)_TRI` (if triangular elements are present). For example, `RGMB_PIN1` and `RMGB_PIN1_TRI`, or `RGMB_ASSEMBLY1` and `RGMB_ASSEMBLY1_TRI`, or `RGMB_CORE` and `RGMB_CORE_TRI`, for workflows ending at the pin, assembly or core level, respectively. 
The default outer boundary names in the radial dimension are `outer_(level)_(type)`. For example, for workflows ending at the pin, assembly or core level respectively, the outer boundary names are `outer_pin_1`, `outer_assembly_1` and `outer_core`. If the problem is extruded in the axial dimension, the  `top` and `bottom` boundaries also exist.

Reply:
@aaronoaks - please use Shikhar's update here. Only the !param things aren't appearing right.

Reply:
Yea they render weird here, but if you "Quote reply" the comment it will recall the raw source which is correct and can be copy/pasted.

@eshemon should this also be added as a slide in the presentation?

Reply:
@aaronoaks  - yes - please add as a slide in the presentation